01_fibonacci/main_test.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestPrintNumber(t *testing.T) {
```

Comment:
Add corner cases as well: `-1, 0` and maybe a very large number that causes and overflow.

Reply:
Added edge cases